packages/react-components/react-toolbar/src/components/ToolbarRadioButton/__snapshots__/ToolbarRadioButton.test.tsx.snap

```diff
@@ -3,9 +3,11 @@
 exports[`ToolbarRadioButton renders a default state 1`] = `
 <div>
   <button
+    aria-checked="false"
```

Comment:
not sure if this part of code is causing it, but I can see in the DOM
both attributes available on the "radio" . 

But when we have "radio", then we need only "aria-checked" not aria-pressed. (https://www.w3.org/TR/wai-aria-1.1/#aria-checked)"
![image](https://user-images.githubusercontent.com/22620150/211768854-235f15e4-45a6-4388-89a2-b006166e8928.png)

Reply:
Yeah the culprit is here https://github.com/microsoft/fluentui/pull/26275/files#r1066774241